ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/phase0/statetransition/epoch/RewardsAndPenaltiesCalculatorPhase0.java

```diff
@@ -166,8 +169,9 @@ public void applyInactivityPenaltyDelta(
       final UInt64 baseReward,
       final UInt64 finalityDelay,
       final RewardAndPenalty delta) {
-
     if (isInactivityLeak(finalityDelay)) {
+      LOG.info("Beacon chain has entered inactivity leak");
```

Comment:
We probably want to say: "Beacon chain is in activity leak" or something like that. "entered" only makes sense for the first time it happens.